scheduler/src/cook/mesos/api.clj

```diff
@@ -754,10 +748,10 @@
 
 (defn- instance->instance-map
   "Converts the instance to a map of relevant fields that will be returned to the caller"
-  [db fid instance]
+  [db framework-id retrieve-url-path-fn instance]
```

Comment:
Is there a way to do this without having to pass around `retrieve-url-path-fn`? It can be annoying to navigate around code when you have to keep going back up to components to find the implementation.